components/org.wso2.carbon.identity.oidc.session/src/test/java/org/wso2/carbon/identity/oidc/session/OIDCSessionStateTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.identity.oidc.session;
+
+import org.powermock.modules.testng.PowerMockTestCase;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.testng.Assert.assertNotNull;
+
+public class OIDCSessionStateTest extends PowerMockTestCase {
+
+    private static final String USERNAME = "user1";
+    private static final String CLIENT_ID_VALUE = "3T9l2uUf8AzNOfmGS9lPEIsdrR8a";
+    private OIDCSessionState oidcSessionState;
+
+    @BeforeTest
+    public void setUp() {
+        oidcSessionState = new OIDCSessionState();
+    }
+
+    @Test
+    public void testSetAuthenticatedUser() {
+        oidcSessionState.setAuthenticatedUser(USERNAME);
+        assertNotNull(oidcSessionState.getAuthenticatedUser());
+    }
+
+    @Test
+    public void testSetSessionParticipants() {
+        Set<String> authenticatedUesrs = new HashSet<>();
+        authenticatedUesrs.add(CLIENT_ID_VALUE);
+        oidcSessionState.setSessionParticipants(authenticatedUesrs);
+        assertNotNull(oidcSessionState.getSessionParticipants());
+    }
+
+    @Test
+    public void testAddSessionParticipant() {
+        String client_id = "ES9l2uUf8AzNOfmGS9lPEIsdrR8a";
+        oidcSessionState.addSessionParticipant(client_id);
+        Set sessionParticipants = oidcSessionState.getSessionParticipants();
+        assertNotNull(sessionParticipants.contains(client_id), "Client_id is a session participant");
+    }
+
+    @Test
+    public void testSetAuthenticated() throws Exception {
+        oidcSessionState.setAuthenticated(true);
+        assertNotNull(oidcSessionState.isAuthenticated());
```

Comment:
you can use assertTrue()  if oidcSessionState.isAuthenticated() returns a boolean